tools/documentation/pkgdown.yml

```diff
@@ -158,6 +158,7 @@ reference:
     desc: Functions that are intended to be called by third-party packages that extend Shiny.
     contents:
       - createWebDependency
+      - registerThemeDependency
```

Comment:
Hopefully no one will have to know this function exists (it's `@keywords internal`). The reason it's exported is so that `bootstraplib::bs_dependency_defer()` can use it under-the-hood

Reply:
Ah, ok. Then I'ma change `documentation` logic. Currently since it's exported, it's forcing a pkgdown entry.

Reply:
I've removed `registerThemeDependency` from the public docs. Also updated the error message to tell us where to make the fixes